test/e2e/common/pods.go

```diff
@@ -27,7 +27,7 @@ import (
 
 	"golang.org/x/net/websocket"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
ditto

Reply:
This is due to the plugin check mechanism of vscode.